packages/chevrotain/src/parse/parser/traits/recognizer_engine.ts

```diff
@@ -817,6 +814,9 @@ export class RecognizerEngine {
     fullName: string,
     idxInCallingRule: number
   ): void {
+    if (typeof idxInCallingRule !== "number") {
```

Comment:
1, how could it ever not be a number?
2. I'd rather not add a type check for **every** Grammar rule invocation as this is a hotspot

Reply:
You are right, I forgot to remove this debugging statment.